docs/api/crud_nodes.md.old

```diff
@@ -0,0 +1,275 @@
+# Node CRUD Operations
```

Comment:
Why check in a file with a .old extension?

Reply:
It's more fun that way.